src/pe_source/data/sixgill/source.py

```diff
@@ -16,6 +16,13 @@
     org_assets,
 )
 
+# Setup logging to central file
+logging.basicConfig(
+    filename="pe_reports_Logging.log",
```

Comment:
How come you are not using the `CENTRAL_LOGGING_FILE` variable here? Also why are you using `filemode="a"` and different formatting options in `src/pe_reports/__init__.py`, but not here?

Reply:
@dav3r 
I have updated the logging filename and formatting to match throughout the project per comit [7ccf5c9](https://github.com/cisagov/pe-reports/pull/169/commits/7ccf5c9c137ff89f5102120ec00d4ba80ff2c71d) I use the filemode='a' to append the file and as I understand this is what the mode that we want so that we have a continually growing log file. Do you have a recommendation as to another filemode that we should be using?

Reply:
👍  (see https://github.com/cisagov/pe-reports/pull/169#discussion_r952796014 for response).

Reply:
@cduhn17 - it looks like this file was overlooked with these changes:

```
from pe_reports import CENTRAL_LOGGING_FILE

# Setup logging to central file
logging.basicConfig(
    filename=CENTRAL_LOGGING_FILE,
    filemode="a",
    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
    datefmt="%m/%d/%Y %I:%M:%S",
    level="INFO",
)

LOGGER = logging.getLogger(__name__)
```

Reply:
@dav3r ,
Updated per commit [a1a9665]([a1a9665](https://github.com/cisagov/pe-reports/pull/169/commits/a1a96655fa2b923e9e630003354e704a2004d758))

Reply:
I still see this:
https://github.com/cisagov/pe-reports/blob/a1a96655fa2b923e9e630003354e704a2004d758/src/pe_source/data/sixgill/source.py#L20-L24

instead of:
```
logging.basicConfig(
    filename=CENTRAL_LOGGING_FILE,
    filemode="a",
    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
    datefmt="%m/%d/%Y %I:%M:%S",
    level="INFO",
)
```

Reply:
@dav3r , 

I have updated this per commit [75cfd8d](https://github.com/cisagov/pe-reports/pull/169/commits/75cfd8d9c5f96a9c80ca7cadd57a069abdbf3176)

Reply:
I think that's the wrong commit, but after https://github.com/cisagov/pe-reports/pull/169/commits/a0631b77b2fce6ad697fdcc1c55b479e83e735ee, this situation is resolved.